src/markdown.tsx

```diff
@@ -13,7 +13,7 @@ import { RootRenderer } from "./renderers/root";
 import { Theme, defaultTheme } from "./themes";
 import { Styles, mergeStyles } from "./themes/themes";
 
-const parser = unified().use(remarkParse).use(remarkGfm);
+const defaultRemarkPlugins: PluggableList = [remarkParse, remarkGfm];
```

Comment:
I think you shouldn't include `remarkParse` in the default plugins and should rather use it by default because
1. here, users can pass an empty array, which will lead to a runtime error (`TypeError: Cannot "processSync" without "Parser"`)
2. `remarkParse` is the official remark parser so users will always want to use it when customizing the plugins list so using it by default would reduce the boilerplate

This would give 
```suggestion
const defaultRemarkPlugins: PluggableList = [remarkGfm];
```